po/it.po

```diff
@@ -1356,9 +1353,8 @@ msgstr ""
 "argomenti specialistici:"
 
 #: src/running-the-course/course-structure.md:19
-#, fuzzy
 msgid "Rust in Android"
-msgstr "Benvenuto in Rust su Android"
+msgstr "Rust su Android"
```

Comment:
optional: I think "Rust in Android" sounds better, but I'm fine also with the current version